src/main/java/net/fabricmc/fabric/mixin/loot/MixinLootEntries.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016, 2017, 2018 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.loot;
+
+import net.fabricmc.fabric.impl.loot.LootEntryRegistryImpl;
+import net.minecraft.world.loot.entry.LootEntries;
+import net.minecraft.world.loot.entry.LootEntry;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(LootEntries.class)
+public class MixinLootEntries {
+	@Shadow
+	private static void register(LootEntry.Serializer<?> lootEntry$Serializer_1) {}
+
+	@Inject(method = "<clinit>", at = @At("RETURN"))
+	private static void onClinit(CallbackInfo info) {
+		LootEntryRegistryImpl.setRegisterFunction(MixinLootEntries::register);
```

Comment:
Does using a lambda like such work? :thinking: Just curious

Reply:
Yes, lambdas in mixins work, I tested it a while ago.

Reply:
It actually does. You can test it by running the test mod.

Reply:
Does it work in non-dev environments?